portal-web/test/functional/com/liferay/portalweb/dependencies/test-theme.war/css/main.css

```diff
@@ -5590,5 +5590,8 @@ you can use the generic selector below, but it's slower:
 /* This file allows you to override default styles in one central location for easier upgrade and maintenance. */
 body {
   background-color: orange; }
+  
+.blue-background.blue-background {
+  background-color: blue !important; }
```

Comment:
From the cadmin confluence page: `Just a note: It's not impenetrable; !important, 2 id's with N classes, or inline styles will beat it.`